tinyiiod.c

```diff
@@ -157,8 +157,8 @@ void tinyiiod_do_write_attr(struct tinyiiod *iiod, const char *device,
 	char buf[128];
```

Comment:
please remove the` ,` from the end of the commit message

Reply:
"libtinyiiod" prefix was added
PR had 3 commits, now it has 6 :)....: this was my mistake trying to solve a conflict, fixed now
PR description is empty, please add it, summarizing the commits.: I will add this
libtinyiiod: Fix out of bounds array access: this is another pull request